src/Microsoft.AspNetCore.Razor.Tasks/DotnetToolTask.cs

```diff
@@ -115,11 +115,11 @@ public override void Cancel()
             string commandLineCommands,
             out int result)
         {
-            CompilerServerLogger.Log("Server execution started.");
+            Log.LogMessage("Server execution started.");
```

Comment:
This should be Low, it's just tracing

Reply:
Just found out that the StandardOutputLoggingImportance is Low. So going to leave these as is.

Reply:
What does that mean? Does that mean that `Log.LogMessage(...)` applies a default of Low?

If that's the case then either remove Low where we specify it, or specify it everywhere. When you do something other than the default, I assume you have a good reason 😆

Reply:
That is the plan. I realized this after sending this update. I'll make it consistent.

Reply:
BTW the default is Normal https://github.com/Microsoft/msbuild/blob/master/src/Shared/TaskLoggingHelper.cs#L292

There are two things that matter:
1. What is the level of the **message**
2. What is the level of the **filter** applied to messages

1 is controlled where you do logging
2 is controlled by `StandardOutputLoggingImportance`

The correct level for the kind of messages that appear here is Low. They are diagnostic tracing and not for end users to see.

Reply:
Wow I didn't expect them to not use https://github.com/Microsoft/msbuild/blob/master/src/Utilities/ToolTask.cs#L376. Okay I'll be explicit.